tools/ubuntu-setup/README.md

```diff
@@ -0,0 +1,45 @@
+# Setting up OpenWhisk on Ubuntu server
+
+The following are verified to work on Ubuntu 14.04.3 LTS. You may need `sudo` or root access to install required software depending on your system setup.
+
+  ```
+  # Install git if it is not installed
+  apt-get install git
+
+  # Clone openwhisk
+  git clone https://github.com/openwhisk/openwhisk.git
+
+  # Install all required software
+  (cd openwhisk/tools/ubuntu-setup && source all.sh)
+  ```
+
+### Select one type of data store when creating vm
+Follow instructions [tools/db/README.md](../db/README.md) on how to configure a a datastore for OpenWhisk
+
+### Build and Deploy
+
+```
+# Build and deploy
+  ant clean build deploy
+  
+# Or teardown and deploy
+  ant redeploy
+```
+To teardown OpenWhisk and remove all Docker containers, run `vagrant ssh -- ant teardown`. You can then redeploy the system with `ant deploy`. To do both at once, use `ant redeploy`.
```

Comment:
fixed in https://github.com/openwhisk/openwhisk/pull/360/commits/a8f784d644c9303bf80d66038f054a4c1be891ca